.github/workflows/extra_tests.yml

```diff
@@ -38,7 +38,9 @@ jobs:
       # sccache doesn't work on incrementally compiled crates, and incremental
       # compilation doesn't help us here anyway.
       CARGO_INCREMENTAL: false
-      # SCCACHE_RECACHE: 1 # Uncomment this to clear cache, then comment it back out
+      # Increment to force eviction of previous caches.
+      # Keep in sync with same variable in `run_tests.yml`.
+      SCCACHE_CACHE_VERSION: 1
```

Comment:
Add a comment about when you might want to do this?

Reply:
done